wger/core/tests/test_register_user_api.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.models import User
+from wger.core.tests.api_base_test import ApiPostTestCase, ApiBaseTestCase
+from wger.core.tests.base_testcase import WorkoutManagerTestCase
+
+
+class ApiUserRegistrationTests(WorkoutManagerTestCase, ApiBaseTestCase, ApiPostTestCase):
+    """
+    Test the API endpoint for registration registration
+    of users
+    """
+    url = '/api/v2/user/register/'
+    url_detail = '/api/v2/user/register/1/'
+    resource = User
+    private_resource = True
+    special_endpoints = ()
+    data = {'username': 'giddy254', 'email': 'giddy@example.com', 'password': 'password'}
```

Comment:
The part of the code for creating user profiles was not being run because some imports had not been made, after making the relevant imports, all the code was run by tests.